js/client/modules/@arangodb/testing.js

```diff
@@ -119,6 +120,7 @@ let optionsDocumentation = [
 ];
 
 const optionsDefaults = {
+  'dumpAgencyOnError': false,
```

Comment:
```suggestion
  'dumpAgencyOnError': true,
```

Reply:
Let's merge it with the dumps turned on by default first. MaxN needs the agency output.
Everyone can turn off the dumps locally if required.
But it is good to have this option now!